config/environments/development.rb

```diff
@@ -2,7 +2,7 @@
 
 Rails.application.configure do
   # Verifies that versions and hashed value of the package contents in the project's package.json
-  config.webpacker.check_yarn_integrity = true
+  config.webpacker.check_yarn_integrity = false
```

Comment:
Thi should probably be restored once the PR is solid... For more information: https://github.com/rails/webpacker/blob/master/README.md#yarn-integrity

Reply:
👍

Reply:
On second thought this seems like an anti-pattern. 

The rails container/service should be concerned with running the rails dev server. The webpack service should be concerned with yarn integrity.

Reply:
There is no "webpack service" though :D webpacker is a library for Rails, this checks if the yarn.lock and the package.json are aligned, the same way Bundler does for Gemfile and Gemfile.lock. I wouldn't worry about it too much. 

It's also the default of the library...

Reply:
Blocked by this:

- https://github.com/rails/webpacker/issues/1568
- https://github.com/yarnpkg/yarn/issues/3167

Works fine when ` config.webpacker.check_yarn_integrity = false`